packages/web/package.json

```diff
@@ -15,8 +15,10 @@
     "isomorphic-unfetch": "^3.0.0",
     "next": "latest",
     "next-google-fonts": "^1.2.1",
+    "next-images": "^1.4.1",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "react-google-font-loader": "^1.1.0",
```

Comment:
is this package in use in `web`?

Reply:
Nope. I had added it to make fonts work. Will remove it